list/pop_back.md

```diff
@@ -0,0 +1,43 @@
+# pop_back
+
+**Description :** The pop_back() function removes an element from the back of a list container. When used, it decreases the size of the container by 1.
+
+**Example** :
+```cpp
+#include <iostream>
+#include <list>
+
+int main(){
+    //declare an empty list
+    std::list<int>mylist; 
```

Comment:
`std::list<int> mylist`

Reply:
Thank you Delta456. As soon as I figure out how to accept changes, I shall :)

Reply:
Light just went on - I'll make the changes and do another PR.